flake.nix

```diff
@@ -1,124 +1,26 @@
-          (builtins.elemAt m 0)
-          replacement
-          (builtins.elemAt m 1)
-        ];
-      production-deps = project.overrideAttrs (oldAttrs: {
-        name = "cardano-sdk-production-deps";
-        configurePhase =
-          builtins.replaceStrings
-          ["yarn install --immutable --immutable-cache"]
-          ["yarn workspaces focus --all --production"]
-          oldAttrs.configurePhase;
-        # A bunch of deps build binaries using node-gyp that requires Python
-        PYTHON = "${pkgs.python3}/bin/python3";
-        # node-hid uses pkg-config to find sources
-        buildInputs = oldAttrs.buildInputs ++ [pkgs.pkg-config pkgs.libusb1];
-
-        installPhase = ''
-          runHook preInstall
-          mkdir -p $out/libexec $out/bin
-          # Move the entire project to the output directory.
-          mv $PWD "$out/libexec/$sourceRoot"
-          cd "$out/libexec/$sourceRoot"
-          # Invoke a plugin internal command to setup binaries.
-          yarn nixify install-bin $out/bin
-          runHook postInstall
-        '';
-      });
-    in
-      project.overrideAttrs (oldAttrs: {
-        # A bunch of deps build binaries using node-gyp that requires Python
-        PYTHON = "${pkgs.python3}/bin/python3";
-        # chromedriver wants to download the binary
-        CHROMEDRIVER_FILEPATH = "${chromedriverBin}";
-        # node-hid uses pkg-config to find sources
-        buildInputs = oldAttrs.buildInputs ++ [pkgs.pkg-config pkgs.libusb1];
-        # run actual build
-        buildPhase = ''
-          yarn workspace @cardano-sdk/cardano-services run build
-        '';
-        # override installPhase to only install what's necessary
-        installPhase = ''
-          runHook preInstall
-
-          mkdir -p $out/libexec/$sourceRoot $out/bin
-          yarn workspace @cardano-sdk/cardano-services-client run build:version
-
-          cp -r ${production-deps}/libexec/$sourceRoot/node_modules $out/libexec/$sourceRoot/node_modules
-          cp -r scripts $out/libexec/$sourceRoot/scripts
-          for p in cardano-services cardano-services-client core ogmios util crypto projection projection-typeorm util-rxjs; do
-            mkdir -p $out/libexec/$sourceRoot/packages/$p
-            cp -r packages/$p/dist $out/libexec/$sourceRoot/packages/$p/dist
-            cp -r packages/$p/package.json $out/libexec/$sourceRoot/packages/$p/package.json
-          done
-          cp -r ${production-deps}/libexec/$sourceRoot/packages/cardano-services/config $out/libexec/$sourceRoot/packages/cardano-services/config
-
-          cd "$out/libexec/$sourceRoot"
+  inputs = {
+    std.url = "github:divnix/std"; # keep minor version in lockstep with GH std-action
+    # std.inputs.nixpkgs.follows = "nixpkgs";
+    std.inputs.n2c.follows = "n2c";
+    haumea.url = "github:nix-community/haumea/v0.2.2";
+    n2c.url = "github:nlewo/nix2container";
+    n2c.inputs.nixpkgs.follows = "std/nixpkgs";
```

Comment:
Let's have separate nixpkgs entry and set other inputs to follow it in hopes of reducing developers mental overhead

Reply:
Addressed in 93cd451b3bf08ad132da0e18839473efda672658